presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/credential/CredentialProviderModule.java

```diff
@@ -15,18 +15,19 @@
 
```

Comment:
Commit message looks invalid. You are not using methods, but modules.

Reply:
This is using [@Provides methods](https://github.com/google/guice/wiki/ProvidesMethods) (which live inside modules) rather than a `Provider`. I've always called these "provider" methods but I'll update this to be `@Provides` since that matches the Guice documentation.

Reply:
Actually, I see your point, this change is about both using `@Provides` methods and switching to explicit modules, so I'll update the message to reflect that. It previously used implicit/inline modules with providers.